site/content/docs/groups/_index.md

```diff
@@ -40,11 +40,63 @@ The screenshot below gives a quick overview of the groups' key features.
     5. CI/CD triggers
 3. **Run all checks in one go** with a configurable concurrency.
 4. Tweak checks in the inline "mini editor" to **quickly build up a group of similar checks**
-5. Use a **common base URL** for your API checks    
+5. Use a **common base URL** for your API checks
+6. **Manage individual checks**, including viewing, editing, and duplicating checks
 
 > [!WARNING]
 > Alert settings being controlled at group level means that a check that is part of a group that has no connected alert channels *will not alert*.
 
+## Creating a check group
```

Comment:
@guolau this first overview section is IMO a bit too much. For other constructs we are not driving users through the whole creation experience, i.e. "click here, then enter this data..." Let's assume that's something the users will get to already, cut away the fat and go straight into explaining the various components of groups as you have done below.